e2e/docker-compose.yaml

```diff
@@ -13,7 +13,7 @@ services:
       - ipfs
       - postgres
     environment:
-      postgres_host: postgres:5432
+      postgres_host: postgres
```

Comment:
Do you know why this is needed all of a sudden?

Reply:
https://github.com/graphprotocol/graph-node/commit/abcc7ee44b4c54da6e271a5173811bb4e2779074

They introduced postgres_port as another argument (defaulting to 5432)